content/webapp/pages/search/index.tsx

```diff
@@ -276,61 +366,85 @@ const NewSearchPage: NextPageWithLayout<NewProps> = ({
           </Container>
           <GridContainer>
             <CatalogueResults>
-              <div>
-                {catalogueResults.works &&
-                  catalogueResults.works.totalResults > 0 && (
-                    <div>
-                      <SectionTitle sectionName="Catalogue" />
-                      {catalogueResults.works.workTypeBuckets?.map(
-                        (bucket, i) => (
-                          <NextLink
-                            key={i}
-                            {...worksLink(
-                              {
-                                query: queryString,
-                                workType: [bucket.data.id],
-                              },
-                              `works_workType_${pathname}`
-                            )}
-                            passHref
-                          >
-                            {bucket.data.label} ({bucket.count})
-                          </NextLink>
-                        )
-                      )}
-                      <NextLink
-                        {...worksLink(
-                          {
-                            query: queryString,
-                          },
-                          `works_all_${pathname}`
-                        )}
-                        passHref
-                      >
-                        All Catalogue results{' '}
-                        {catalogueResults.works?.totalResults}
-                      </NextLink>
-                    </div>
+              {(!!(
+                catalogueResults.images?.totalResults &&
+                catalogueResults.images?.totalResults > 0
+              ) ||
+                !!(
+                  catalogueResults.works?.totalResults &&
+                  catalogueResults.works?.totalResults > 0
+                )) && (
```

Comment:
I _think_ `catalogueResults.images?.totalResults > 0 || catalogueResults.works?.totalResults > 0` could be simplified equivalent? (`undefined > 0 = false`)

Reply:
Ok I hadn't tested it so possible! I just thought it was a bit complex to read but it's not hard to understand what it actually does, so np